indico/modules/rb/operations/bookings.py

```diff
@@ -156,9 +156,10 @@ def get_room_candidates(candidates, conflicts):
             if not (any(candidate.overlaps(conflict) for conflict in conflicts))]
 
 
-def _bookings_query(filters):
+def _bookings_query(filters, noload_room=False):
     reservation_strategy = contains_eager('reservation')
-    reservation_strategy.noload('room')
+    if noload_room:
+        reservation_strategy.noload('room')
```

Comment:
I'd rather use `raiseload` so it fails loudly if there's code accessing it anyway.